Classes/Controller/Backend/ControlCenter/TemplaVoilaPlus8UpdateController.php

```diff
@@ -441,14 +441,14 @@ protected function checkAllPageContentForTo(array $validatedToWithDs): array
                 if (isset($validatedToWithDs[$row['tx_templavoilaplus_to']])) {
                     $validatedToWithDs[$row['tx_templavoilaplus_to']]['countUsage'] += $row['COUNT(`uid`)'];
                 } else {
-                    $validationErrors[] = 'There are pages which use an non existent Template Object with uid "' . $row['tx_templavoilaplus_to'] . '" like page with page uid: "' . $row['uid'] . '"';
+                    $validationErrors[] = 'There is a page using a non-existent Template Object with uid "' . $row['tx_templavoilaplus_to'] . '" like page with page uid: "' . $row['uid'] . '"';
```

Comment:
Hmmm, there is minimum one page, we have no count here so don't know if it is one ore more